src/main/java/dinkplugin/notifiers/LevelNotifier.java

```diff
@@ -126,15 +136,24 @@ private void attemptNotify() {
             .build());
     }
 
-    private boolean checkLevelInterval(int level) {
+    private boolean checkLevelInterval(int previous, int level) {
         if (level < config.levelMinValue())
             return false;
+
         if (level > 99 && !config.levelNotifyVirtual())
             return false;
+
         int interval = config.levelInterval();
-        return interval <= 1
-            || level == 99
-            || level % interval == 0;
+        if (interval <= 1 || level == 99)
+            return true;
+
+        // Check levels in (previous, current] for divisibility by interval
+        // Allows for firing notification if jumping over a level that would've notified
+        for (int lvl = level; lvl > previous; lvl--) {
```

Comment:
Do we care to do this smarter using the output of the modulo check to make this $O(1)$ ?